src/main/java/mate/academy/spring/controller/AuthenticationController.java

```diff
@@ -0,0 +1,29 @@
+package mate.academy.spring.controller;
+
+import mate.academy.spring.mapper.DtoResponseMapper;
+import mate.academy.spring.model.User;
+import mate.academy.spring.model.dto.request.UserRequestDto;
+import mate.academy.spring.model.dto.response.UserResponseDto;
+import mate.academy.spring.security.AuthenticationService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AuthenticationController {
+    private final AuthenticationService authenticationService;
+    private final DtoResponseMapper<UserResponseDto, User> userResponseMapper;
+
+    public AuthenticationController(AuthenticationService authenticationService,
+                                    DtoResponseMapper<UserResponseDto, User> userResponseMapper) {
+        this.authenticationService = authenticationService;
+        this.userResponseMapper = userResponseMapper;
+    }
+
+    @PostMapping("/register")
+    public UserResponseDto register(@RequestBody UserRequestDto userRequestDto) {
```

Comment:
Don't you forget to add 'Valid' annotation here ?

Reply:
Yes, I did 😞